From 0a7579dbfdb50f75d69a781b7895268620694c67 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=E9=BB=84=E6=B6=9B?= <huangtao@rock-chips.com>
Date: Sat, 4 Dec 2010 14:27:38 +0800
Subject: [PATCH] magnetometer: fix
 drivers/staging/iio/magnetometer/Kconfig:5:warning: multi-line strings not
 supported

---
 drivers/staging/iio/magnetometer/Kconfig | 45 ++++++++++++------------
 1 file changed, 22 insertions(+), 23 deletions(-)

diff --git a/drivers/staging/iio/magnetometer/Kconfig b/drivers/staging/iio/magnetometer/Kconfig
index 7d023296e810..973396e1ddfa 100644
--- a/drivers/staging/iio/magnetometer/Kconfig
+++ b/drivers/staging/iio/magnetometer/Kconfig
@@ -1,25 +1,24 @@
-
 #
 # Magnetometer sensors
-#s"
-comment "Magnetometer sensor
-config SENSORS_AK8975                                          
-  tristate "Asahi Kasei AK8975 3-Axis Magnetometer"             
-  default n                                                     
-  depends on I2C                                                
-  help                                                          
-  Say yes here to build support for Asahi Kasei AK8975 3-Axis   
-  Magnetometer.                                                 
-                                                                
-  To compile this driver as a module, choose M here: the module 
-  will be called ak8975.      
-config SENSORS_AK8973                                          
-  tristate "Asahi Kasei AK8973 3-Axis Magnetometer"             
-  default n                                                     
-  depends on I2C                                                
-  help                                                          
-  Say yes here to build support for Asahi Kasei AK8973 3-Axis   
-  Magnetometer.                                                 
-                                                                
-  To compile this driver as a module, choose M here: the module 
-  will be called ak8973.                                        
\ No newline at end of file
+#
+
+comment "Magnetometer sensors"
+config SENSORS_AK8975
+  tristate "Asahi Kasei AK8975 3-Axis Magnetometer"
+  depends on I2C
+  help
+  Say yes here to build support for Asahi Kasei AK8975 3-Axis
+  Magnetometer.
+
+  To compile this driver as a module, choose M here: the module
+  will be called ak8975.
+
+config SENSORS_AK8973
+  tristate "Asahi Kasei AK8973 3-Axis Magnetometer"
+  depends on I2C
+  help
+  Say yes here to build support for Asahi Kasei AK8973 3-Axis
+  Magnetometer.
+
+  To compile this driver as a module, choose M here: the module
+  will be called ak8973.
-- 
2.35.3

